inc/metabox/extend-metabox/src/settings.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Meta Options build.
+ */
+ import { PluginSidebar, PluginSidebarMoreMenuItem } from '@wordpress/edit-post';
+ const { __ } = wp.i18n;
+ const { Component, Fragment} = wp.element;
+ const { compose } = wp.compose;
+ const { withSelect, withDispatch } = wp.data;
+
+ class AstraMetaSettings extends Component {
```

Comment:
```suggestion
 class MetaSettings extends Component {
```
@sagarb90 Let's convert this to functional components.